src/devices/StUsb4500/Helper/UsbPdObjectHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace Iot.Device.Usb.Helper
```

Comment:
same, we only use namespaces when otherwise it would cause collisions or mess (i.e. Spi/I2c code touches similar concepts but differ slightly) - in here it's all about USB and therefore it should be in the same namespace (if we decide to have these helpers also for other bindings then you could consider changing it)